app/models/container_template.rb

```diff
@@ -0,0 +1,15 @@
+class ContainerTemplate < ApplicationRecord
+  include CustomAttributeMixin
+
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
```

Comment:
Not sure you need both ExtManagementSystem and ContainerProject associations, project is already connected to provider, so you can keep just the association to the project.